### ISSUE TITLE
Send both a multiple-float OSC message and multiple single-float messages

### ISSUE THREAD

**Issue by simondemeule** [Xmodal/autonomX#173]
As a user, I want to receive both a multiple-float OSC message and multiple single-float messages.

## Criteria
- Send all output values as a list with the address `/generator_name/output f f f ...`
- Send all output values as atomic messages with the address `/generator_name/output/output_number f'` (where output_number corresponds to the number of the output as shown on the interface)

## Technical details
Pseudo-code:
```
for each generator
  send one OSC message with a list of floats
  for each output region
    send one OSC message with a single float
```

**Comment by aalex** [comment]
It's often difficult to choose between start URL-based message format vs key-value arguments, etc. We should take a few minutes, and discuss it with @sofian and @alx-s .

**Comment by alx-s** [comment]
Good point but not a priority. Unless we run into issues and re-evaluate, we will stick to the list message implemented in release 0.1

**Comment by aalex** [comment]
![image](https://user-images.githubusercontent.com/293677/126222231-47606d94-463b-4da5-9340-9f266ef9517d.png)
